device/iot-device-client/src/main/java/com/microsoft/azure/sdk/iot/device/DeviceIO.java

```diff
@@ -164,14 +164,36 @@ void open() throws IOException
         }
     }
 
-    void registerMultiplexedDeviceClient(List<DeviceClientConfig> configs) throws InterruptedException
+    // Functionally the same as "open()", but without wrapping any thrown TransportException into an IOException
+    void multiplexingClientOpen() throws TransportException
```

Comment:
Would feel more natural to me if the verb were first: `openMultiplexingClient`. Thoughts?

Reply:
This doesn't open the multiplexing client though. It is the open method used exclusively by the multiplexing client

Reply:
I'll rename this to something like "openWithoutWrappingException" instead, so that this isn't necessarily limited to multiplexing clients though

Reply:
I don't know how someone would know that.

Reply:
And I'll do the same for the similar close method